lib/src/components/NxNexusPageHeader/NxNexusPageHeader.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { HeaderLinkProps, Props, propTypes } from './types';
+
+export { Props };
+
+import './NxNexusPageHeader.scss';
+
+const logoImg = require('../../assets/img/logo_nexus_generic.svg');
+
+function HeaderLink({ name, href, current }: HeaderLinkProps) {
+  const classes = classnames('nx-page-header__link', {
+    'nx-page-header__link--current': current
+  });
+
+  return (
+    <a className={classes} href={href} data-text={name}>{name}</a>
+  );
+}
+
+// visible for testing
+export { HeaderLinkProps, HeaderLink };
+
+export default function NxNexusPageHeader({ links, homeLink, productInfo, children }: Props) {
+  const logoEl = <img src={logoImg} className="nx-product__logo-image" alt="⬡ Sonatype"/>,
+      linkEls = links && links.map(link => <HeaderLink key={link.name} { ...link } />);
+
+  return (
+    <header className="nx-page-header nx-page-header--nexus">
+      <div className="nx-page-header__inner">
+        <div className="nx-product">
+          <div className="nx-product__logo">
+            { homeLink ? <a className="nx-product__link" href={homeLink}>{logoEl}</a> : logoEl }
+          </div>
+          <div className="nx-product__top-row">
```

Comment:
How about `nx-product__summary` for this one and `nx-product__details` for the lower one?

Reply:
> How about `nx-product__summary` for this one and `nx-product__details` for the lower one?

Works for me: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/fb3ae07c988230b25e10d6b58146a8e517236ec0